newIDE/app/src/GameDashboard/LeaderboardAdmin/index.js

```diff
@@ -150,12 +152,38 @@ const getApiError = (payload: LeaderboardUpdatePayload): ApiError => ({
   ),
 });
 
+const getExtremeAllowedScoreText = (
+  i18n: I18nType,
+  currentLeaderboard: Leaderboard
+) => {
+  const maxOrMin =
+    currentLeaderboard.sort === 'ASC' ? i18n._(t`min`) : i18n._(t`max`);
+  if (
+    currentLeaderboard.extremeAllowedScore === null ||
+    currentLeaderboard.extremeAllowedScore === undefined
+  ) {
+    return '';
+  }
+  let customizedScoreText = currentLeaderboard.extremeAllowedScore;
+  if (currentLeaderboard.customizationSettings) {
+    customizedScoreText = formatScore(
+      currentLeaderboard.extremeAllowedScore,
+      currentLeaderboard.customizationSettings.scoreFormatting
+    );
+  }
+  return ` (${maxOrMin}: ${customizedScoreText})`;
```

Comment:
Some languages may not use this construction "`(max: 150)`" but instead write thing like `(150: max)` or even not use colon/parenthesis.

So it's better to put a `Trans` or `i18n._(t...)` on a larger sentence, like the 4 possibilities:

Higher is better
Higher is better (max : X)
Lower is better
Higher is better (min : X)